config/metric_config_sample.yaml

```diff
@@ -36,6 +36,8 @@ output:
       saveMode: Overwrite
       # path of the file (can be omitted if using tableName and writing to warehouse)
       path: file.parquet
+      # Optional: If enabled, adding current timestamp suffix to path of the file
+      createUniquePath: true
```

Comment:
maybe call it `addTimestampToPath` ?

Reply:
@RonBarabash I think that's how Irena originally called it :) I asked her to call it createUniquePath cause I thought it would be more comprehensible. The ts doesn't really matter here, it's more about the versioning option and not running over stuff, don't you agree?